lib/posix-sysinfo/sysinfo.c

```diff
@@ -51,6 +51,15 @@
 #include <vfscore/file.h>
 #endif
 
+/**
+ * The Unikraft `struct utsname` structure.
+ *
+ * * `sysname` and `nodename` are set to "unikraft".
```

Comment:
Does it matter that sysname is set to Unikraft (with capital U) and nodename is set to unikraft (without capital u)?